opensrp-opd/src/main/java/org/smartregister/opd/presenter/BaseOpdRegisterActivityPresenter.java

```diff
@@ -117,9 +134,17 @@ public void startForm(@NonNull String formName, @NonNull String entityId, @NonNu
 
         JSONObject form = null;
         try {
+            OpdDiagnosisAndTreatmentForm opdDiagnosisAndTreatmentForm = new OpdDiagnosisAndTreatmentForm();
```

Comment:
You are not checking for the form type here before performing these operations? I am assuming that this is a general method?